### PR TITLE
Use hooks in README and add style import

### PR DESCRIPTION
**Pull request by nagayev**
### Motivation
Copy paste example didn't works

### CODE REVIEW

**Comment by nagayev** [comment]
@satya164 ping

File: README.md [hunk 1]
```diff
@@ -44,20 +44,14 @@ import Editor from 'react-simple-code-editor';
...
-  return a + b;
-}
-`;
-
-class App extends React.Component {
-  state = { code };
-
-  render() {
+function App(){
+    const [code,setCode] = React.useState(`function add(a, b) {return a + b; }`);
```
Comment (caub) [inline README.md:50]: can you format the code missing spaces), `[code,setCode]`, `function App() {`

Comment (caub) [reply README.md:50]: even `function add(a, b) {\n  return a + b;\n}` too

Comment (caub) [reply README.md:50]: fix indentation too `    const [code,setCode]` should be 1 indent level, 2 spaces

Comment (nagayev) [reply README.md:50]: @caub 
Okey, I'll fix it later.

Comment (nagayev) [reply README.md]: @caub 
I have fixed codestyle.
Can you merge this PR, please?

File: README.md [hunk 2]
```diff
@@ -39,27 +39,29 @@ yarn add react-simple-code-editor
...
+import Editor from "react-simple-code-editor";
+import { highlight, languages } from "prismjs/components/prism-core";
+import "prismjs/components/prism-clike";
+import "prismjs/components/prism-javascript";
+import "prismjs/themes/prism.css"; //Example style, you can use another
+
+function App() {
+  const [code, setCode] = React.useState(`function add(a, b) {
+    return a + b;
+     }`);
```
Comment (caub) [inline README.md:52]: shouldn't it be:
```js
  const [code, setCode] = React.useState(`function add(a, b) {
  return a + b;
}`)
```
If you want to multiline it,else
```js
  const [code, setCode] = React.useState(`function add(a, b) {\n  return a + b;\n}`)
```

**Pull request closed without merging**